executor/src/witgen/machines/machine_extractor.rs

```diff
@@ -57,25 +57,26 @@ pub fn split_out_machines<'a, T: FieldElement>(
             base_identities.iter().cloned().partition(|i| {
                 // The identity has at least one machine witness, but
```

Comment:
Can you update this documentation? It does not fit any more to the fact that we only consider the 'left' part.
The idea is that lookups from that machine are part of the machine, but not those "to" the machine, right?

Reply:
At the same time, the implementation of `all_row_connected_witnesses` should consider all witnesses on the left hand side of a lookup to be row-connected but also all witnesses on the right hand side of a lookup (but they sides are not row-connected), right?

Reply:
Sure, I updated the comment and elaborated a bit. Also, yes, `all_row_connected_witnesses` should work exactly as you describe.